js/main.js

```diff
@@ -57,9 +55,9 @@ function addEntry(entry) {
   img.setAttribute('src', entry.url);
   titleH2.textContent = entry.title;
   notesP.textContent = entry.notes;
-  if (entry === entriesObj) {
-    ul.prepend(entryLi);
-  }
+
+  var newEntry = ul.appendChild(entryLi);
```

Comment:
You shouldn't be appending to the `ul` here at all, just returning the DOM tree.